department-of-veterans-affairs/caseflow
# Moved server error display to Add Hearing modal.

Resolves #8496

### Description
Moved the persistence of a new hearing day to the add hearing modal to display the server errors in the modal itself.

### Acceptance Criteria

### Testing Plan
1. Go to Add Hearing Date
2. Enter a Central hearing for a particular date and save it.
3. Enter a Central hearing for the same date
4. Confirm you get an error message displayed in the modal after clicking on confirm.
